ert/storage/_storage.py

```diff
@@ -710,11 +710,11 @@ def get_experiment_responses(*, experiment_name: str) -> Iterable[str]:
         )
 
     ensemble_id = experiment["ensemble_ids"][0]  # currently just one ens per exp
-    response = _get_from_server(f"ensembles/{ensemble_id}")
+    response = _get_from_server(f"ensembles/{ensemble_id}/responses")
 
     if response.status_code != 200:
         raise ert.exceptions.StorageError(response.text)
-    return list(response.json()["response_names"])
+    return list(response.json().keys())
```

Comment:
What are the values here that we are just discarding?

Reply:
I can enrich it via a comment ....

Reply:
That is very nice! I was just wondering and making sure that we didn't download the actual data unnecessary